<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="46924" started-at="2021-06-27T15:05:28Z" finished-at="2021-06-27T15:06:15Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="46924" started-at="2021-06-27T15:05:28Z" finished-at="2021-06-27T15:06:15Z">
      <class name="com.qa.tests.LoginPageTest">
        <test-method status="PASS" signature="setExtent()[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="setExtent" is-config="true" duration-ms="156" started-at="2021-06-27T15:05:28Z" finished-at="2021-06-27T15:05:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setExtent -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="setup" is-config="true" duration-ms="10415" started-at="2021-06-27T15:05:28Z" finished-at="2021-06-27T15:05:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="verifyLoginHeader()[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="verifyLoginHeader" duration-ms="92" started-at="2021-06-27T15:05:38Z" finished-at="2021-06-27T15:05:38Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [HOME] but found [Home]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [HOME] but found [Home]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.qa.tests.LoginPageTest.verifyLoginHeader(LoginPageTest.java:80)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginHeader -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="tearDown" is-config="true" duration-ms="995" started-at="2021-06-27T15:05:38Z" finished-at="2021-06-27T15:05:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginHeader status=FAILURE method=LoginPageTest.verifyLoginHeader()[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="setup" is-config="true" duration-ms="7300" started-at="2021-06-27T15:05:39Z" finished-at="2021-06-27T15:05:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="LoginPageTitleTest()[pri:1, instance:com.qa.tests.LoginPageTest@36f6e879]" name="LoginPageTitleTest" duration-ms="7" started-at="2021-06-27T15:05:47Z" finished-at="2021-06-27T15:05:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPageTitleTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="tearDown" is-config="true" duration-ms="684" started-at="2021-06-27T15:05:47Z" finished-at="2021-06-27T15:05:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginPageTitleTest status=SUCCESS method=LoginPageTest.LoginPageTitleTest()[pri:1, instance:com.qa.tests.LoginPageTest@36f6e879] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="setup" is-config="true" duration-ms="6951" started-at="2021-06-27T15:05:48Z" finished-at="2021-06-27T15:05:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="LoginTest(java.lang.String, java.lang.String)[pri:2, instance:com.qa.tests.LoginPageTest@36f6e879]" name="LoginTest" duration-ms="6094" started-at="2021-06-27T15:05:55Z" data-provider="getLoginData" finished-at="2021-06-27T15:06:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[user4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[test4]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [CRMPRO] but found [Free CRM - CRM software for customer relationship management, sales, and support.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [CRMPRO] but found [Free CRM - CRM software for customer relationship management, sales, and support.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.qa.tests.LoginPageTest.LoginTest(LoginPageTest.java:73)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="tearDown" is-config="true" duration-ms="1102" started-at="2021-06-27T15:06:01Z" finished-at="2021-06-27T15:06:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginTest status=FAILURE method=LoginPageTest.LoginTest(java.lang.String, java.lang.String)[pri:2, instance:com.qa.tests.LoginPageTest@36f6e879] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="setup" is-config="true" duration-ms="7196" started-at="2021-06-27T15:06:02Z" finished-at="2021-06-27T15:06:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="LoginTest(java.lang.String, java.lang.String)[pri:2, instance:com.qa.tests.LoginPageTest@36f6e879]" name="LoginTest" duration-ms="4273" started-at="2021-06-27T15:06:09Z" data-provider="getLoginData" finished-at="2021-06-27T15:06:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[rohanw]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[John@028]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="tearDown" is-config="true" duration-ms="702" started-at="2021-06-27T15:06:14Z" finished-at="2021-06-27T15:06:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginTest status=SUCCESS method=LoginPageTest.LoginTest(java.lang.String, java.lang.String)[pri:2, instance:com.qa.tests.LoginPageTest@36f6e879] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="closeExtent()[pri:0, instance:com.qa.tests.LoginPageTest@36f6e879]" name="closeExtent" is-config="true" duration-ms="275" started-at="2021-06-27T15:06:14Z" finished-at="2021-06-27T15:06:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeExtent -->
      </class> <!-- com.qa.tests.LoginPageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
